05/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zadanie 5</title>
+    <link rel="stylesheet" href="./styles/base.css">
+    <link rel="stylesheet" media="(max-width: 599px)" href="./styles/header-mobile.css">
+    <link rel="stylesheet" media="(max-width: 599px)" href="./styles/content-mobile.css">
+    <link rel="stylesheet" media="(min-width: 599px)" href="./styles/content.css">
+    <link rel="stylesheet" href="./styles/footer.css">
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 599px)" href="./styles/tablet.css">
+    <link rel="stylesheet" media="(min-width: 599px)" href="./styles/header-desktop.css">
+    <link rel="stylesheet" media="(min-width: 1199px)" href="./styles/desktop.css">
+</head>
+
+<body>
+
+
+    <!-- Header start -->
+
+    <header class="header row">
+        <div class="header__container container">
+            <h1 class="header__logo cell"></h1>
+            <nav class="header__nav cell">
+            <nav class="header__nav--mobile cell">
+                <input type="checkbox" id="menu__mobile--hamburger" />
+ 
+                <label class="hamburger" for="menu__mobile--hamburger"></label>
+            </nav>    
+            </nav>
+        </div>
+    </header>
+
+    <!-- Moduł 1 -->
+
+    <section class="module--1 content row">
```

Comment:
Raczej staramy się używać nazw słownie zamiast samego `--1` czy `--2` (czasami trzeba zmienić ich kolejność i wtedy numerki nie będą po kolei, co może wprowadzać w błąd.

PS. Jeśli używamy BEM to zawsze Modyfikator występuje z Elementem lub Blockiem czyli tutaj powinno być tak: `module module--1`. Tutaj znajdziesz więcej informacji: https://devmentor.pl/b/metodologia-bem-w-css-i-sass